packages/types/lib/auth/http.api.ts

```diff
@@ -40,7 +39,6 @@ export type PostPublicTableauAuthorization = Endpoint<{
     };
     Querystring: {
         connection_id?: string | undefined;
-        public_key: string;
```

Comment:
isn't the actual fix public_key + session_token ?
Also I see it was missing in PostPublicUnauthenticatedAuthorization

Reply:
we can add them both but they are never used in the endpoint logic so decided not to have them. wdyt?

Reply:
yeah I see your point, ideally, we have something always correct but typescript is too lax on partial objects. no need to fix 👍🏻